app/models/company.rb

```diff
@@ -6,6 +6,9 @@ class Company < ActiveRecord::Base
   validates :title,           presence: true
   validates :description,     presence: true
   validates :about,           presence: true
-  validates :address,         presence: true
+  validates :address1,        presence: true
+  validates :city,            presence: true
+  validates :state,           presence: true
```

Comment:
@arafatm Added a new validation to make sure only 2 capital letters are used for state